app/__init__.py

```diff
@@ -11,5 +11,5 @@ def create_app():
     database.init_app(app)
     migrations.init_app(app)
     routes.init_app(app)
-
+    auth.init_app(app)
```

Comment:
I believe this `auth.init_app(app)` should be before the `routes`, because we use the content from the auth inside the routes.